packages/react-sdk/src/index.tsx

```diff
@@ -115,18 +115,19 @@ export function BucketProvider({
       sdkVersion: SDK_VERSION,
     });
     clientRef.current = client;
+
+    client.onFeaturesUpdated(() => {
```

Comment:
Can we provide an effect hook for this `useFeaturesUpdatedEffect()`?

Reply:
hmm, yes. What's the use-case?

Reply:
Oh, ignore me. I thought this change was in an example. It's actually in the SDK itself.